CHANGES.md

```diff
@@ -8,6 +8,8 @@
 * #219 - reduced search graph for configuration dlls to avoid native assets
 * #221 - added support for conditional/leveled enrichers from Serilog 2.9+
 * #222 - updated Microsoft.Extensions.DependencyModel
+* #237 - DependencyContextAssemblyFinder fix: check `serilog` in any part of the name for any dependent package
```

Comment:
Not true now:
> return library.Dependencies.Any(dependency =>
                    dependency.Name.StartsWith(Serilog, StringComparison.OrdinalIgnoreCase) &&
                   (dependency.Name.Length == Serilog.Length || dependency.Name[Serilog.Length] == '.'));